File: spec/code_samples/Python_python3/profiles@documents/patch.py
import http.client

conn = http.client.HTTPSConnection("www.directfreight.com")

payload = "{\"current_name\":\"<ADD STRING VALUE>\",\"new_name\":\"<ADD STRING VALUE>\"}"

conn.request("PATCH", "/api/profiles/documents", payload)

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))

File: spec/code_samples/Php_http1/profiles@documents/patch.php
<?php

HttpRequest::methodRegister('PATCH');
$request = new HttpRequest();
$request->setUrl('https://www.directfreight.com/api/profiles/documents');
$request->setMethod(HttpRequest::HTTP_METH_PATCH);

$request->setBody('{"current_name":"<ADD STRING VALUE>","new_name":"<ADD STRING VALUE>"}');

try {
  $response = $request->send();

  echo $response->getBody();
} catch (HttpException $ex) {
  echo $ex;
}